apps/web/cypress/support/commands.ts

```diff
@@ -137,7 +138,8 @@ Cypress.Commands.add('editDuplicateJob', (job: IENApplicantJobCreateUpdateDTO) =
 });
 
 Cypress.Commands.add('addMilestone', (milestone: IENApplicantAddStatusDTO) => {
-  cy.get('#start_date').focus().click().type(`${milestone.start_date}`);
+  cy.get('#start_date').focus().clear();
+  cy.get('#start_date').focus().clear().type(`${milestone.start_date}`);
```

Comment:
how come we duplicate focus and clear actions on same field?

Reply:
I don't know. I tried several ways but always got 'detached dom' error.